comment/tests/test_api/test_views.py

```diff
@@ -320,6 +320,18 @@ def get_base_url(self, reaction, c_id=None):
             c_id = self.comment.id
         return reverse_lazy('comment-api:react', args=[c_id, reaction])
 
+    def test_cannot_update_comment_content(self):
+        comment = self.comment
+        original_content = comment.content
+        data = {'content': 'test updation during reactions'}
+
+        response = self.client.post(self.get_base_url(self.like), data=data, content_type='application/json')
```

Comment:
What is the response data? Why is the statusCode 200? Shouldn't it be 400?

Reply:
the status code is 200 because as designed currently we have made the content field read only. So, the changes are ignored but no errors are thrown.

the response data is the one returned for a successful reaction on a comment.

Reply:
i have also added the test for response data as well.